client/src/tests/procedures_tests.rs

```diff
@@ -92,7 +92,9 @@ async fn usecase_ed25519() {
 
     {
         use crypto::signatures::ed25519::{PublicKey, Signature};
-        let pk = PublicKey::from_compressed_bytes(pk).unwrap();
+
+        // api change crypto.rs 0.5 -> 0.7
```

Comment:
IMO the comments `// api change crypto.rs 0.5 -> 0.7` are more confusing then helpful for a user. The historic implementation is not relevant, and there is no knowledge gain from this information. I would suggest to instead list the relevant changes in a comment on this PR / in the CHANGELOG file, and remove all these type of comments.